sdk/src/main/java/com/mapbox/maps/MapInitOptions.kt

```diff
@@ -24,6 +25,7 @@ data class MapInitOptions constructor(
   var mapOptions: MapOptions = getDefaultMapOptions(context),
   var initialCameraOptions: CameraOptions? = null,
   var textureView: Boolean = false,
+  val styleUri: String? = Style.MAPBOX_STREETS,
```

Comment:
Probably we also want to add this field to the xml attributes?

Reply:
Yes, will add it.